src/components/RelicUnlockModal.js

```diff
@@ -81,15 +88,65 @@ export default function RelicUnlockModal({ isOpen, setIsOpen, tier }) {
             </div>
           ))}
         </div>
+      </>
+    );
+
+  const renderReloadedOptions = () => {
+    const reloadedRelics = [
+      ...RELICS[0].filter(relic => existingRelics[0] !== relic.id),
+      ...RELICS[1].filter(relic => existingRelics[1] !== relic.id),
+      ...RELICS[2].filter(relic => existingRelics[2] !== relic.id),
```

Comment:
Probably could just use `RELICS[3]` (t4) and check for `relic => relic.tier !== tier` instead of this but it works